dataparser/data_parser/data_parser.py

```diff
@@ -182,17 +206,20 @@ def _get_library_kw_source(self, libcode, keyword):
         func_file = None
         if hasattr(libcode, kw_func):
             func = getattr(libcode, kw_func)
+        else:
+            func_file, func = None, None
         if func:
-            kw_class = self.get_class_that_defined_method(func)
-            if kw_class:
-                func_file = self.get_function_file(kw_class)
-            else:
-                func_file = self.get_function_file(func)
+            # kw_class = self.get_class_that_defined_method(func)
+            # if kw_class:
+            #     func_file = self.get_function_file(kw_class)
+            # else:
+            #     func_file = self.get_function_file(func)

```

Comment:
If this is not needed, I would delete the commented lines.

Reply:
Removed